src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6591,4 +6591,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_UnreadRecordParameter_Title" xml:space="preserve">
     <value>Parameter is unread. Did you forget to use it to initialize the property with that name?</value>
   </data>
+  <data name="ERR_RecordAmbigCtor" xml:space="preserve">
+    <value>Synthesized copy constructor conflicts with a primary constructor.</value>
```

Comment:
nit: consider wording the message as something like "Primary constructor is not allowed to use the copy constructor signature 'Rec(Rec)'", for example. The primary constructor is the bit that needs to change, so should be the first to be mentioned, and showing the SymbolDisplay I think provides a bit of a hint as to what about the primary constructor needs to change.

Reply:
Makes sense to mention the primary constructor first. Adjusted

---
In reply to: [544674566](https://github.com/dotnet/roslyn/pull/49975#discussion_r544674566) [](ancestors = 544674566)